client/components/tinymce/style.scss

```diff
@@ -1,9 +1,9 @@
-@import 'plugins/wplink/style';
-@import 'plugins/calypso-alert/style';
-@import 'plugins/insert-menu/style';
-@import 'plugins/wpcom-help/style';
-@import 'plugins/wpcom-charmap/style';
-@import 'plugins/contact-form/style';
+@import 'plugins/wplink/wplink';
```

Comment:
Can't we leave the files named `style.scss` and remove these imports, letting discovery find them instead?